.env.example

```diff
@@ -112,6 +112,7 @@ SMTP_SERVER=smtp.gmail.com
 SMTP_PORT=587
 SMTP_AUTHENTICATION=plain
 SMTP_ENABLE_STARTTLS_AUTO=true
+SMTP_SSL=true
```

Comment:
This should be false, otherwise it would change the default of new installations.